install/build_development_apache.sh

```diff
@@ -77,7 +77,7 @@ cd "$third_party/apr/src"
 ./buildconf --prefix=$TARGET
 ./configure --prefix=$TARGET
 make
-make install
+MAKEFLAGS="" make install
```

Comment:
This fails if -j is set in MAKEFLAGS? Surely a better solution would be to just invoke make with -j 1